src/routes/chat/SocketService/socketSetup.js

```diff
@@ -0,0 +1,15 @@
+import io from 'socket.io-client';
+
+const Socket = {
+  socket: null,
+  connect: null,
+  load: null
+};
+
+const connect = () => {
+  Socket.socket = io.connect(`${process.env.REACT_APP_BACKEND_API}chat`);
+};
+
+connect();
```

Comment:
Not sure if this is good, since the socket will automatically connect once this file is imported. Maybe force users to call it manually?